DQM/Integration/python/test/visualization-live_cfg.py

```diff
@@ -41,6 +35,18 @@
 process = scenario.visualizationProcessing(globalTag='DUMMY', writeTiers=['FEVT'], **kwds)
 
 process.source = source
+process.source.inputFileTransitionsEachEvent = cms.untracked.bool(True)
+process.source.skipFirstLumis = cms.untracked.bool(True)
+
+m = re.search(r"\((\w+)\)", str(source.runNumber))
+runno = str(m.group(1))
+outDir= '/fff/BU0/output/EvD/run'+runno
+
+create output directory
```

Comment:
Hi @danduggan,
This causes a syntax error: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9067/60/build.log. I guess you missed the "#".